query-creator/README.md

```diff
@@ -212,19 +214,19 @@ Next, you can use AWS Serverless Application Repository to deploy ready to use A
       username string,
       student_name string,
       teachers array<struct<user_id: string, name: string, district: string, state: string, email: string>>,
-      last_run timestamp
+      last_run string
```

Comment:
@scytacki and I had some discussion earlier about making this a timestamp rather than a string.  I'm assuming adding something to the table in their timestamp format wasn't straightforward?

Reply:
I spent a bit of time trying to cast this in the query, but was unsuccessful. I think because by the time it was reading it in the query it was already too late, it was complaining that `learner.timestamp` was supposed to be a timestamp but wasn't.

I am not exactly sure if there was something I was missing (that's partly why I extracted this to its own commit)

Reply:
Let's leave it as a string. A timestamp would allow us to do more advanced queries, but there are bigger fish to fry.